user_docs/en/userGuide.t2t

```diff
@@ -1089,6 +1089,8 @@ Once recognition is complete, the result will be presented in a document similar
 Pressing enter or space will activate (normally click) the text at the cursor if possible.
 Pressing escape dismisses the recognition result.
 
+When you want to monitor constantly changing content, such as when watching a video with sub titles, you can optionally enable automatic refresh of the recognized content in the [Windows OCR category #Win10OcrSettings] of the [NVDA Settings #NVDASettings] dialog.

```

Comment:
The refresh interval is now bound to the UWP provider.

Reply:
Actually, that was not my point since I had not noticed this, but this fully makes sense to have the refresh interval specifically linked to the UWP OCR provider and not to all providers.

My point was just regarding documentation, to move this sentence ("When you want to monitor ... of the [NVDA Settings #NVDASettings] dialog."). This sentence is currently in the general paragraph "Content Recognition" and I suggest to move it to the paragraph "Windows OCR", just below.

Sorry for my unclear previous comment with a useless suggestion.